tests/plugins/test_google.py

```diff
@@ -140,55 +121,490 @@ def execute(self):
+
+
+def test_load_credentials_from_file_valid_with_z_expiry() -> None:
+    """Test loading credentials with Z-suffixed expiry
+
+    Generated by Claude 3.5 Sonnet to improve test coverage.
+    """
+    cred_data = {
+        'access_token': 'test_token',
+        'refresh_token': 'test_refresh_token',
+        'token_uri': 'https://oauth2.googleapis.com/token',
+        'client_id': 'test_client_id',
+        'client_secret': 'test_client_secret',
+        'scopes': ['https://www.googleapis.com/auth/calendar.readonly'],
+        'token_expiry': '2025-12-31T23:59:59Z'
+        }
+
+    with tempfile.NamedTemporaryFile(mode='w', suffix='.json', delete=False) as f:
+        json.dump(cred_data, f)
+        f.flush()
+        try:
+            result = google.load_credentials_from_file(f.name)
+            assert result is not None
+            assert result.token == 'test_token'
+            assert result.refresh_token == 'test_refresh_token'
+        finally:
+            os.unlink(f.name)
+
+
+def test_load_credentials_from_file_valid_without_z_expiry() -> None:
+    """Test loading credentials without Z-suffixed expiry
+
+    Generated by Claude 3.5 Sonnet to improve test coverage.
+    """
+    cred_data = {
+        'access_token': 'test_token',
+        'refresh_token': 'test_refresh_token',
+        'token_uri': 'https://oauth2.googleapis.com/token',
+        'client_id': 'test_client_id',
+        'client_secret': 'test_client_secret',
+        'scopes': ['https://www.googleapis.com/auth/calendar.readonly'],
+        'token_expiry': '2025-12-31T23:59:59'
+        }
+
+    with tempfile.NamedTemporaryFile(mode='w', suffix='.json', delete=False) as f:
+        json.dump(cred_data, f)
+        f.flush()
+        try:
+            result = google.load_credentials_from_file(f.name)
+            assert result is not None
+            assert result.token == 'test_token'
+            assert result.refresh_token == 'test_refresh_token'
+        finally:
+            os.unlink(f.name)
+
+
+def test_load_credentials_from_file_invalid_expiry() -> None:
+    """Test loading credentials with invalid expiry format
+
+    Generated by Claude 3.5 Sonnet to improve test coverage.
+    """
+    cred_data = {
+        'access_token': 'test_token',
+        'refresh_token': 'test_refresh_token',
+        'token_uri': 'https://oauth2.googleapis.com/token',
+        'client_id': 'test_client_id',
+        'client_secret': 'test_client_secret',
+        'scopes': ['https://www.googleapis.com/auth/calendar.readonly'],
+        'token_expiry': 'invalid_date_format'
+        }
+
+    with tempfile.NamedTemporaryFile(mode='w', suffix='.json', delete=False) as f:
+        json.dump(cred_data, f)
+        f.flush()
+        try:
+            result = google.load_credentials_from_file(f.name)
+            assert result is not None
+            assert result.expiry is None  # Should be None due to ValueError
+        finally:
+            os.unlink(f.name)
```

Comment:
Where's the difference in behaviour here? The `token_expiry` seems to have no effect on the `try` part. Also, is it important that `2025-12-31T23:59:59` lies in the future? If so I highly recommend to generate this date. Otherwise we're having a not so happy new-year :)